dislib/recommendation/als/base.py

```diff
@@ -0,0 +1,250 @@
+from math import sqrt
+
+import numpy as np
+from pycompss.api.api import compss_wait_on
+from pycompss.api.task import task
+from scipy import sparse
+from sklearn.metrics import mean_squared_error
+
+from dislib.data import load_data
+
+
+class ALS(object):
+    """ Alternating Least Squares recommendation.
+
+    Implements distributed alternating least squares recommendation based on
+    Zhou et al. [1]_.
+
+    Parameters
+    ----------
+    max_iter : int, optional (default=100)
+        Maximum number of iterations to perform.
+    tol : float, optional (default=1e-4)
```

Comment:
It says default value is 1e-4 but it is set to 0.0001 ** 2 which is 1e-8?

Reply:
right, I missed the **2